css/style.css

```diff
@@ -303,15 +303,16 @@ html {
   background-size: cover;
   width: 100%;
   padding: 10rem 0;
-  padding-left: 81.5rem;
+  display: flex;
+  justify-content: flex-end;
 }
 
 .dark-square {
   width: 57rem;
-  height: 55.8rem;
   background-color: rgba(0, 0, 0, 0.6);
   padding: 7.6rem;
   text-align: center;
+  margin-right: 21.5rem;
```

Comment:
It's expected to be responsibility of the container padding, as I pointed above

Reply:
Ok